WordPress/Classes/ViewRelated/Post/AztecPostViewController.swift

```diff
@@ -319,7 +336,7 @@ extension AztecPostViewController
     }
 }
 
-
+// MARK: -
```

Comment:
Is this intentional?

When looking at the complete file in Xcode it feels a bit arbitrary.